packages/@uppy/server-utils/src/Provider.js

```diff
@@ -17,7 +17,7 @@ module.exports = class Provider extends RequestClient {
   }
 
   get defaultHeaders () {
-    return Object.assign({}, super.defaultHeaders, {'uppy-auth-token': localStorage.getItem(this.tokenKey)})
+    return Object.assign(super.defaultHeaders, {'uppy-auth-token': localStorage.getItem(this.tokenKey)})
```

Comment:
this change is correct but it looked wrong initially :thinking: maybe add a comment explaining that super.defaultHeaders is a getter, so it's fine to mutably merge things into it? else i'm sure we'll get a PR to add back the `{}, ` in the near future :)

(FWIW i think keeping the `{}, ` is also fine, one or two empty objects don't pressure the GC that much :smile: )